web/src/pages/HomePage/HomePage.tsx

```diff
@@ -49,75 +49,77 @@ const HomePage = () => {
     <>
       <Metadata title="Home" description="Home page" />
 
-      <main className="flex min-h-screen w-full flex-wrap items-center justify-center bg-primary">
+      <main className="min-h-screen w-full bg-primary">
```

Comment:
It looks like the new changes are shifting the design upwards. Is this intended? 

Now:

![image](https://github.com/user-attachments/assets/27e4da2b-e7b6-4560-bd37-1bac746cc323)


Before:
![image](https://github.com/user-attachments/assets/8da09909-6091-4d8e-b94a-eccac1559197)

Reply:
🤔